tests/test_backends_clifford.py

```diff
@@ -250,6 +247,7 @@ def test_bitflip_noise(backend):
 
 
 def test_set_backend(backend):
+    set_backend(backend.name)
```

Comment:
Does this work for the `qibojit` backends as well?

Reply:
Apparently it works, but it's actually not needed. I'll update this test.